examples/functions-with-required-parameters/functions_with_required_parameters.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+// This function accepts three `int` values, performs a calculation
+// and returns the result as an integer.
+// All three parameters are required parameters.
+function calculate(int a, int b, int c) returns int {
+    return a + 2 * b + 3 * c;
+}
+
+public function main() {
+    // Call the `calculate()` function to retrieve the integer result.
+    // Arguments for required parameters can be passed as positional arguments.
+    // Positional arguments need to be passed in the expected order.
+    int result = calculate(5, 6, 7);
+    // Print the result.
+    io:println(result);
+
+    // Arguments for required parameters can also be passed as named arguments.
+    // All arguments after the first named argument need to be passed by name.
+    // Named arguments do not have to be specified in order.
```

Comment:
```suggestion
    // Named arguments do not have to be specified in any order.
```

Reply:
Do we need this change?

IMO, any makes sense in a context like "you can specify them in any order".

But here, we are trying to say explicitly that you don't have to specify them in order, since specifying them in order is the usual way.